sdk-extensions/aws/src/test/java/io/opentelemetry/sdk/extension/aws/trace/XraySamplerClientTest.java

```diff
@@ -10,45 +10,24 @@
 import static org.assertj.core.api.Assertions.entry;
 
 import com.google.common.io.ByteStreams;
+import com.google.common.io.Resources;
+import com.linecorp.armeria.common.AggregatedHttpRequest;
 import com.linecorp.armeria.common.HttpResponse;
 import com.linecorp.armeria.common.HttpStatus;
 import com.linecorp.armeria.common.MediaType;
-import com.linecorp.armeria.server.ServerBuilder;
-import com.linecorp.armeria.server.logging.LoggingService;
-import com.linecorp.armeria.testing.junit5.server.ServerExtension;
+import com.linecorp.armeria.testing.junit5.server.mock.MockWebServerExtension;
+import java.nio.charset.StandardCharsets;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Date;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
+import org.skyscreamer.jsonassert.JSONAssert;
 
 class XraySamplerClientTest {
```

Comment:
Shouldn't we also add a malformed response test? Just to verify that in such case an unchecked IO exception will be always thrown.